dev_guide/deployments/advanced_deployment_strategies.adoc

```diff
@@ -96,90 +117,215 @@ $ oc patch route/bluegreen-example -p '{"spec":{"to":{"name":"example-blue"}}}'
 
 . In your browser, refresh the page until you see the *v2* image.
 
+
 [[advanced-deployment-a-b-deployment]]
 == A/B Deployment
 
-A/B deployments generally imply running two (or more) versions of the
-application code or application configuration at the same time for testing or
-experimentation purposes.
+The A/B xref:deployment_strategies.adoc#strategies[deployment strategy] lets you 
+try a new version of the application in a limited way in the production
+environment. You can specify that the production version gets most of the user requests
+while a limited fraction of requests go to the new version. Since you control the 
+portion of requests to each version, as testing progresses you can increase the 
+fraction of requests to the new version and ultimately stop using the previous version.
+As you adjust the request load on each version the number of pods in each service
+may need to be scaled as well to provide the expected performance.
 
-The simplest form of an A/B deployment is to divide production traffic between
-two or more distinct *shards* -- a single group of instances with homogeneous
-configuration and code.
+In addition to upgrading software, this feature can be used to experiment with 
+versions of the user interface. Since some users get the old version and some the new,
+you can evaluate the user's reaction to the different versions to inform design decisions.
 
-More complicated A/B deployments may involve a specialized proxy or load
```

Comment:
You deleted the bit about more complex strategies for splitting the traffic... I think it is useful to keep that and say that the product itself doesn't support it, but that it could be done by exposing two routes and splitting with an external service.